### ISSUE TITLE
Composer exception - could not load package

### ISSUE THREAD

**Issue by JanStiegler** [short-pixel-optimizer/enable-media-replace#66]
it seems that some fields like `name` are missing in the [composer.json](https://github.com/short-pixel-optimizer/enable-media-replace/blob/master/composer.json) which leads to exceptions while trying to install this plugin via composer

```
Reading composer.json of https://github.com/short-pixel-optimizer/enable-media-replace.git (master)
Skipped branch master, Undefined index: name

Reading composer.json of https://github.com/short-pixel-optimizer/enable-media-replace.git (updates)
Skipped branch updates, Undefined index: name

[Composer\Repository\InvalidRepositoryException]
No valid composer.json was found in any branch or tag of https://github.com/short-pixel-optimizer/enable-media-replace.git, could not load a package from it.

```

**Comment by RazvanFarte** [comment]
Hey Jan,

Could you run the composer command once more? I made a PR and I would need your help to confirm it concludes.

**Comment by RazvanFarte** [comment]
The functionality was included, although no feedback was given. Closing for now

**Comment by pdobrescu** [comment]
This should be fixed by #67, @JanStiegler please let us know if you still encounter any issues.
